src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -0,0 +1,304 @@
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item">Block</li>
+    <li class="breadcrumb-item">
+      <a routerLink="/block/rbd">Images</a></li>
+    <li class="breadcrumb-item active">{{ editing ? 'Edit' : 'Add' }}</li>
+  </ol>
+</nav>
+
+<div class="col-sm-12 col-lg-6">
+  <form name="rbdForm"
+        class="form-horizontal"
+        #formDir="ngForm"
+        [formGroup]="rbdForm"
+        novalidate>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">{{ editing ? 'Edit' : 'Add' }} RBD</h3>
```

Comment:
The text is not translated, maybe you could implement it like i did it [for RGW](https://github.com/ceph/ceph/pull/21351/files#diff-cdf6482138ddac78221c4010b241b83cR27).